run/types/testing.ts

```diff
@@ -304,8 +308,21 @@ export type AccessibilityId =
   | '1 week'
   | '2 weeks'
   | 'Off'
+  | '5 seconds - Radio'
+  | '10 seconds - Radio'
+  | '30 seconds - Radio'
+  | '1 minute - Radio'
+  | '5 minutes - Radio'
+  | '30 minutes - Radio'
+  | '1 hour - Radio'
+  | '12 hours - Radio'
+  | '1 day - Radio'
+  | '1 week - Radio'
+  | '2 weeks - Radio'
+  | 'Off - Radio'
```

Comment:
Avoid duplication.

Right before those lines, you have the same lines without the  `- Radio`.
You can reuse a type in another type.
For instance you could have a 
```ts
type TimerOptions = '5 seconds'
  | '10 seconds'	 
  | '30 seconds'	  
  | '1 minute'	 
  | '5 minutes'	
  | '30 minutes'	
  | '1 hour'	
  | '12 hours'	 
  | '1 day'	
  | '1 week'	 
  | '2 weeks'	  
  | 'Off'	  
```

and have 
```ts
type AccessibilityId = 
[..everything else excluding the ones in TimerOptions and the ones with the `- Radio` at the end]
| TimerOptions
| `${TimerOptions} - Radio` // all the values of TimerOptions with a - Radio at the end
```

```